tensorflow/python/kernel_tests/matmul_op_test.py

```diff
@@ -70,25 +70,25 @@ def _GetTransposedMatrices(x, x_name, kwargs):
     return x
 
 class MatMulTest(test_lib.TestCase):
-  pass
+  pass # Filled in below
 
 def _GetMatMulTest(a_np_, b_np_, use_static_shape_, **kwargs_):
 
   @test_util.run_without_tensor_float_32("Tests matmul")
   def Test(self):
-    if a_np_.dtype is dtypes.bfloat16:
+    if a_np_.dtype==dtypes.bfloat16.as_numpy_dtype:
```

Comment:
Nit: Please add a space on each side of `==`, i.e.,
```suggestion
    if a_np_.dtype == dtypes.bfloat16.as_numpy_dtype:
```